package.json

```diff
@@ -61,7 +63,6 @@
     "npm-run-all": "4.1.5",
     "prettier": "3.3.2",
     "ts-jest": "29.1.5",
-    "tsx": "4.15.6",
```

Comment:
Seems unexpected that you removed this because we generally use this as our runtime